src/gamemanager.js

```diff
@@ -33,4 +33,8 @@ class GameManager {
   }
 }
 
-export default GameManager;
+const manager = new GameManager();
+
+export default manager;
+
+export {GameManager};
```

Comment:
Small nit but you can combine the two export lines

```js
export { manager as default, GameManager };
```

Reply:
Fixed by 2ea853e